core/integration/odc/plugin.go

```diff
@@ -1357,128 +1377,6 @@ func (p *Plugin) CallStack(data interface{}) (stack map[string]interface{}) {
 		}
 		return
 	}
-	stack["ConfigureLegacy"] = func() (out string) {
```

Comment:
No, this is a bit of legacy code we haven't been using for over a year, perhaps years. I remove it because we don't need it any more, and I do it as part of this PR because it saves me the trouble of implementing event production for this function.

Reply:
OK, I still think that it should have been separate branch and PR before of this PR, but it is just nitpicking